### ISSUE TITLE
Extempore directs all informative error messages from audio layer to /dev/null

### ISSUE THREAD

**Issue by giorashh** [digego/extempore#160]
```
$ ./extempore --print-devices                              
<...>
audio device[5]:system api[2]:JACK Audio Connection Kit inchan[2] outchan[8]

$ ./extempore --device 5
------------- Extempore -------------- 
Andrew Sorensen (c) 2010-2014
andrew@moso.com.au, @digego

ARCH           : x86_64-unknown-linux-gnu
CPU            : 
ATTRS          : 
Invalid stream pointer
Invalid stream pointer
```

This happens when the sample rate set for JACK does not match extempore's hardcoded 44100.

**Comment by rekado** [comment]
I ran into the same issue and assumed that JACK support was broken.  It would be helpful if a nicer error was printed that mentioned the sample rate settings.

**Comment by benswift** [comment]
Yep, this should print a more informative error. Do you have a patch
handy?  If not, I'll look into it when I have the time.

**Comment by benswift** [comment]
This predates the v0.9.0 overhaul (May 2026) by several years, and the audio layer (PortAudio, device handling) has been reworked since. Closing as part of a tracker clean-up — if the JACK sample-rate mismatch still swallows the informative error messages on the current release, please reopen with a repro and we'll take a look.
